src/main/java/com/marmelab/quoridor/model/Board.java

```diff
@@ -0,0 +1,41 @@
+package com.marmelab.quoridor.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+
+    public static final int DEFAULT_BOARD_SIZE = 9;
+
+    private final int boardSize;
+
+    private List<Node> nodes;
+
+    public Board() {
+        this(DEFAULT_BOARD_SIZE);
+    }
+
+    public Board(int boardSize) {
+        if (boardSize % 2 == 0) {
+            throw new IllegalArgumentException("The size must be an odd number");
+        } else if (boardSize < 0) {
+            throw new IllegalArgumentException("The size must be positive");
+        }
+        this.boardSize = boardSize;
+        this.nodes = new ArrayList<>();
```

Comment:
How comes the parameter type is absent? Do java guess its type from the attribute?